README.md

```diff
@@ -21,6 +21,67 @@ You can deploy Che plugin registry on Openshift with command.
              -p IMAGE_TAG="latest" \
              -p PULL_POLICY="IfNotPresent"
 ```
+## Docker
+```
+docker run -it  -p 8080:8080 eclipse/che-plugin-registry
+```
+
+## Get index list of all plugins
+Example:
+```
+curl  "http://localhost:8080/index.json"
```

Comment:
could it be directly on / ?

Reply:
Not for now. In general  - don't know. Would you like to request this as a new feature?

Reply:
https://github.com/eclipse/che-plugin-registry/issues/26